app/api/probe_dock/metrics_api.rb

```diff
@@ -145,13 +168,7 @@ def count_new_tests_by_interval(start_date, interval)
       get :projectHealth do
         authorize! :organization, :data
 
-
-        # Retrieve the data from the specified version otherwise take the most recent project version to retrieve the data
-        if params[:projectVersionId].present?
-          project_version = ProjectVersion.where('api_id = ?', params[:projectVersionId]).first
-        else
-          project_version = ProjectVersion.joins(:project).where('projects.api_id = ?', params[:projectId]).order('created_at DESC').first
-        end
+        project_version = project_version()
```

Comment:
I would avoid this, it looks very confusing. It's a good idea in general to avoid overriding methods with local variables. (And on a side note, in this case it is not compliant with the style guide as there should be no parentheses for a method call with no arguments.) We could name the method `current_project_version`, as it takes the project version defined by the current params like `current_organization` does. And leave the local variable as is.